snmp/datadog_checks/snmp/data/default_profiles/3com-huawei.yaml

```diff
@@ -32,7 +32,7 @@ metrics:
       name: hwdevMFanStatusTable  # indexes: hwDevMFanNum
       OID: 1.3.6.1.4.1.43.45.1.2.23.1.9.1.1
     symbols:
-      - name: hwdevMFan
+      - name: hwdevMFanStatus
```

Comment:
Update metric name to be more consistent with table name.

Constant metric should be same as Table symbol name without `Table` suffix.

hwdevMFanStatusTable -> hwdevMFanStatus